lib/tasks/import_data.rb

```diff
@@ -0,0 +1,63 @@
+require "json"
+
+module Tasks
+  class ImportData
+    def initialize(file:, total_lines:, stdout:, draft:)
+      @file = file
+      @total_lines = total_lines
+      @stdout = stdout
+      @draft = draft
+    end
+
+    def import_all
+      file.each.with_index(1) do |json, index|
+        parsed_json = JSON.parse(json).deep_symbolize_keys
+
+        updated_at = parsed_json.fetch(:updated_at)
+        content_item_hash = parsed_json.fetch(:content_item)
+        content_id = content_item_hash.fetch(:content_id)
+
+        unless Event.where("payload ->> 'content_id' = '#{content_id}'").exists?
```

Comment:
This isn't quite right; content_ids are not unique by themselves, it is the combination of (content_id, locale) that is unique, so if we do this subsequent locales for an item won't be imported. Also, this is likely to get very slow as the import proceeds. Might it be better to query the derived table?